docs/Run-Cluster-Genesis.rst

```diff
@@ -325,31 +405,33 @@ these rules. An example of a user generated "generic" file would be::
 SSH Keys
 --------
 
-The OpenPOWER Cluster Genesis Software will generate a passphrase-less SSH
+The OpenPOWER POWER-Up Software will generate a passphrase-less SSH
 key pair which is distributed to
 each node in the cluster in the /root/.ssh directory. The public key is
 written to the authorized\_keys file in the /root/.ssh directory and
 also to the /home/userid-default/.ssh directory. This key pair can be
 used for gaining passwordless root login to the cluster nodes or
 passwordless access to the userid-default. On the deployer node, the
-keypair is written to the ~/.ssh directory as id\_rsa\_ansible-generated
-and id\_rsa\_ansible-generated.pub. To login to one of the cluster nodes
+key pair is written to the ~/.ssh directory as gen
+and gen.pub. To login to one of the cluster nodes
 as root from the deployer node::
 
-    ssh -i ~/.ssh/id_rsa_ansible-generated root@a.b.c.d
+    ssh -i ~/.ssh/gen root@a.b.c.d
 
 As root, you can log into any node in the cluster from any other node in
 the cluster as::
 
     ssh root@a.b.c.d
 
-where a.b.c.d is the ip address of the port used for pxe install. These
-addresses are stored under the keyname *ipv4-pxe* in the inventory file.
+where a.b.c.d is the IP address of the port used for pxe install. These
+addresses are stored under the key name *ipv4-pxe* in the inventory file.
 The inventory file is stored on every node in the cluster at
 /var/oprc/inventory.yml. The inventory file is also stored on the
```

Comment:
We only transfer the 'inventory.yml' to a client if it's assigned the 'solution_inventory' role. We do something similar with the _private_ SSH key and the 'solution_keys' role.